parsing/dialects/dialect1/config.fmpp

```diff
@@ -1123,15 +1124,13 @@ data: {
       "InlineCaseSpecific"
       "InlineCaseSpecificNamedFunctionCall"
       "InlineModOperator"
-      "AlternativeTypeConversionLiteralOrIdentifier"
-      "InlineFormatLiteralOrIdentifier"
       "NamedLiteralOrIdentifier"
+      "AlternativeTypeConversionLiteralOrIdentifier"
```

Comment:
nit: Can we order alphabetically? If not, we should probably add a comment saying why.

Reply:
There's some tests that fail if the Named method comes after. I can write a comment to explain why